server/core/src/main/java/org/infinispan/server/core/logging/LogUtil.java

```diff
@@ -0,0 +1,63 @@
+package org.infinispan.server.core.logging;
+
+import static java.time.temporal.ChronoField.DAY_OF_MONTH;
+import static java.time.temporal.ChronoField.HOUR_OF_DAY;
+import static java.time.temporal.ChronoField.MINUTE_OF_HOUR;
+import static java.time.temporal.ChronoField.MONTH_OF_YEAR;
+import static java.time.temporal.ChronoField.SECOND_OF_MINUTE;
+import static java.time.temporal.ChronoField.YEAR;
+
+import java.time.ZoneId;
+import java.time.chrono.IsoChronology;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.ResolverStyle;
+import java.time.format.SignStyle;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Logging utilities for server modules
+ *
+ * @author Tristan Tarrant
+ * @since 9.2
+ */
+
+public class LogUtil {
+
+   public final static DateTimeFormatter LOG_DATE_TIME_FORMATTER;
+   static {
+      Map<Long, String> moy = new HashMap<>();
+      moy.put(1L, "Jan");
+      moy.put(2L, "Feb");
+      moy.put(3L, "Mar");
+      moy.put(4L, "Apr");
+      moy.put(5L, "May");
+      moy.put(6L, "Jun");
+      moy.put(7L, "Jul");
+      moy.put(8L, "Aug");
+      moy.put(9L, "Sep");
+      moy.put(10L, "Oct");
+      moy.put(11L, "Nov");
+      moy.put(12L, "Dec");
+      LOG_DATE_TIME_FORMATTER = new DateTimeFormatterBuilder()
+            .parseCaseInsensitive()
+            .parseLenient()
+            .appendValue(DAY_OF_MONTH, 1, 2, SignStyle.NOT_NEGATIVE)
+            .appendLiteral('/')
+            .appendText(MONTH_OF_YEAR, moy)
```

Comment:
Questions: 
1) in the server config file, there are handlers ```HR-ACCESS-FILE``` and ```REST-ACCESS-FILE``` that apparently allows you to define the ```pattern-formatter``` for the access log, but those are actually ignored in favor of this harcoded format?
2) What about i18 here? Are going to a fixed date format with month names in english?